driver-core/src/test/unit/com/mongodb/internal/connection/InternalStreamConnectionSpecification.groovy

```diff
@@ -452,40 +462,6 @@ class InternalStreamConnectionSpecification extends Specification {
         !connection.isClosed()
     }
 
-    def 'should notify all asynchronous writers of an exception'() {
```

Comment:
This test concurrently writes to a connection despite this is not being allowed by its API. If the test were to use `AsynchronousSocketChannelStream` or `TlsChannelStream` instead of a mocked stream, they would have thrown a `WritePendingException`. I deleted this test because it does not seen to be useful.